app/app.py

```diff
@@ -11,106 +11,51 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import sys
-import os
```

Comment:
I stripped a lot of code out of this file. A lot of it was not valid anymore and it didn't look like an officially supported tool, since there were comments about stuff that needs to be tested (which was out of date anyway).